src/components/ManageLoginData.jsx

```diff
@@ -0,0 +1,19 @@
+import React,{ useState } from "react";
```

Comment:
Please update all your files to use single quotes. Double quotes isn't wrong, but the convention at neslo is single quotes